File: test-automation-core/src/main/java/com/nortal/test/core/services/report/html/ChangedFilesReport.java
package com.nortal.test.core.services.report.html;

import java.util.List;
import lombok.Data;

@Data
public class ChangedFilesReport {
	private final List<Commit> commits;
	private final long sinceDays;

	public int size() {
		return commits.size();
	}

	public String getStats() {
		return String.format("There were %s commits to golden data during last %s days.\n", size(), sinceDays);
	}

}

File: test-automation-core-old/src/main/java/com/nortal/test/core/services/report/html/ChangedFilesReport.kt
package com.nortal.test.core.services.report.html

import lombok.Data

@Data
class ChangedFilesReport {
    private val commits: List<Commit>? = null
    private val sinceDays: Long = 0
    fun size(): Int {
        return commits!!.size
    }

    val stats: String
        get() = String.format("There were %s commits to golden data during last %s days.\n", size(), sinceDays)
}